Upstream-Status: Accepted

Make ghost script work with long building directory

This patch has been accepted by upstream and should be removed
at the next update of ghostscript.

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>

Index: base/genconf.c
==============================================================
--- a/base/genconf.c
+++ b/base/genconf.c
@@ -682,7 +682,7 @@ read_dev(config_t * pconf, const char *arg)
     string_item_t *item;
     const char *in;
 
-#define MAX_TOKEN 256
+#define MAX_TOKEN 512
     char *token = malloc(MAX_TOKEN + 1);
     char *category = malloc(MAX_TOKEN + 1);
     int file_index;
